lua/entities/gmod_wire_egp/lib/egplib/transmitreceive.lua

```diff
@@ -619,19 +619,31 @@ if (SERVER) then
 					DataToSend[#DataToSend+1] = { ID = obj.ID, index = obj.index, Settings = obj:DataStreamInfo() }
 				end
 
-				timer.Simple( k, function() -- send 1 second apart
+				timer.Simple( k - 1, function() -- send 1 second apart, send the first one instantly
```

Comment:
Don't send the first one instantly, there's probably other networking we don't want to collide with.

Reply:
Works fine, otherwise you'll always have a 1s delay for linking up with a egp/getting into a egp chair

Reply:
Fine I guess